src/Tasks/Backup/DbDumperFactory.php

```diff
@@ -47,6 +47,7 @@ public static function createFromConnection(string $dbConnectionName): DbDumper
 
         if ($dbDumper instanceof MySql) {
             $dbDumper
+                ->setSkipSsl($dbConfig['dump']['skipSsl'] ?? false)
```

Comment:
Can you use `skip_ssl` instead as the other config options use underscores, can you also add this to the docs?

Reply:
I will this weekend

Reply:
done, also remember to merge this https://github.com/spatie/db-dumper/pull/213 if not it won't work